packages/plugin/src/theia.d.ts

```diff
@@ -3436,10 +3436,17 @@ export module '@theia/plugin' {
         /**
          * The exit code reported by the shell.
          *
-         * Note that `undefined` means the shell either did not report an exit  code (ie. the shell
-         * integration script is misbehaving) or the shell reported a command started before the command
-         * finished (eg. a sub-shell was opened). Generally this should not happen, depending on the use
-         * case, it may be best to treat this as a failure.
+         * When this is `undefined` it can mean several things:
```

Comment:
These are just drive-by fixes, right?

Reply:
Yes, the comments were changed in equivalent file in vscode, but these are only better explanation on the API behavior.